Dockerfile

```diff
@@ -19,6 +19,7 @@ RUN --mount=type=bind,target=. \
 
 FROM amazonlinux:2 as bin-unix
 COPY --from=build /out/controller /controller
+USER 1002
```

Comment:
why 1002? does this id have to exist or any random id is ok?

Reply:
random ID works too, i picked 1002 since it's same as distroless's non-root user